tests/_test_msui/test_wms_control.py

```diff
@@ -495,12 +495,10 @@ class TestWMSControlWidgetSetupSimple:
         <Extent name="ELEVATION" default="900.0"> 500.0,600.0,700.0,900.0 </Extent>"""
 
     @pytest.fixture(autouse=True)
-    def setup(self, qtbot):
+    def setup(self, tmp_path_factory, qtbot):
```

Comment:
Why `tmp_path_factory` instead of `tmp_path`?

Reply:
I found this problem by investigating timeouts 

https://pytest-with-eric.com/pytest-best-practices/pytest-tmp-path/#How-Many-Test-Runs-Does-Pytest-keep-in-tmp-path

reading about that the tmp_path will be retained I had the impression that tmp_path_factory may be better.
I liked also that I have not to deal with path separators for adding a name I can lookup.

Reply:
> reading about that the tmp_path will be retained I had the impression that tmp_path_factory may be better.

I'd expect both to be the same in this regard. Only the last 3 temporary directories are retained by default.

> I liked also that I have not to deal with path separators for adding a name I can lookup.

If you want to check that the wms_cache is set to the temporary directory, then check that they are equal. Checking for a known part expected in the path is just a poor proxy for that.

Reply:
after this is fixed i have to lookup the md5_filename again. This is also based on the dir of wms_cache and gets a wrong path currently